better-i18n-for-your-angular-apps/proposal.yaml

```diff
@@ -0,0 +1,35 @@
+---
+Session:
+  Title: Better i18n for your Angular apps
+  Description: >
+    Internationalization and Localization are the two fundamental procedures
```

Comment:
Internationalization and Localization are fundamental to making your
    applications available in different languages and cultures. This is fairly
    involved and would ideally have a tightly integrated toolchain and workflow
    to support the entire process. Angular currently only offers some
    rudimentary support.  While the community has built some great extensions
    to help in this area, we would really like to have first class support from
    the framework.  In this talk, Chirayu and Pascal are going to show you
    their roadmap for a to bring a great i18n experience to the Angular framework.